django/db/migrations/executor.py

```diff
@@ -166,6 +173,9 @@ def unapply_migration(self, state, migration, fake=False):
         if not fake:
             with self.connection.schema_editor() as schema_editor:
                 state = migration.unapply(state, schema_editor)
+                logger.info('Unapplied migration %s on database %s' % (migration, self.connection.alias))
```

Comment:
I'd use the term "Rolled back migration" (although good English would be `"Rolled migration %s back on database %s"`)